doc/source/Platforms.rst

```diff
@@ -311,13 +311,13 @@ The following is required for building new spack environments and for using spac
 Amazon Web Services Ubuntu 20.04
 --------------------------------
 
-For ``spack-stack-1.0.0``, use a t2.2xlarge instance or similar with AMI "skylab-1.0.0-ubuntu20". After logging in, run:
+For ``spack-stack-1.0.2``, use a t2.2xlarge instance or similar with AMI "skylab-1.0.0-ubuntu20". After logging in, run:
 
 .. code-block:: console
 
    ulimit -s unlimited
    module use /home/ubuntu/spack-stack-v1/envs/skylab-1.0.0/install/modulefiles/Core
-   module load stack-gcc/10.3.0
+   module load stack-gcc/9.4.0
```

Comment:
This change was prompted by seeing a difference between the ReadTheDocs instructions versus these instructions. I was guessing that ReadTheDocs had the correct version since we were focused on that more recently with the first skylab release.

Reply:
I verified that 9.4.0 is the correct version on Ubuntu.